docs/user-guide/kubectl-overview.md

```diff
@@ -33,160 +33,271 @@ Documentation for other releases can be found at
+**Important**: Flags that you specify from the command line override default values and any corresponding environment variables. *Tip*: Use short flags sparingly and only for the most frequently used options.<br/>
+For more information and examples about specifying flags, see the [kubectl conventions](../devel/kubectl-conventions.md#flag-conventions).
+
+If you need help, just run `kubectl help` from the terminal window.
+
+## Operations
+
+The following table includes short descriptions and the general syntax for all of the `kubectl` operations:
+
+Operation       | Syntax	|       Description
+-------------------- | -------------------- | --------------------
+`annotate`	| `kubectl annotate [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--resource-version=version]` | Add or update the annotations of a resource.
+`api-versions`	| `kubectl api-versions` | List the API versions in use.
+`apply`			| `kubectl apply` | Apply a configuration change to a resource by file or stdin.
+`attach`		| `kubectl attach POD -c CONTAINER` | Attach to a running container to send stdin from the terminal window.
+`cluster-info`	| `kubectl cluster-info` | Display endpoint information about the master and services in the cluster.
+`config`		| `kubectl config SUBCOMMAND` | Modifies kubeconfig files. See the sub-commands for details.
+`create`		| `kubectl create -f FILENAME` | Create a resource using a file or stdin.
+`delete`		| `kubectl delete ([-f FILENAME] | TYPE [(NAME | -l label | --all)])` | Delete resources either by filenames, label selectors, names, resource selectors, resources, or stdin.
+`describe`	| `kubectl describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME)` | Display all the details for one or more resources.
+`edit`		| `kubectl edit (RESOURCE/NAME | -f FILENAME)` | Edit the definition of a resource on the server.
+`exec`		| `kubectl exec POD [-c CONTAINER] -- COMMAND [args...]` | Execute a command against a container in a pod.
+`expose`		| `kubectl expose (-f FILENAME | TYPE NAME) [--port=port] [--protocol=TCP|UDP] [--target-port=number-or-name] [--name=name] [----external-ip=external-ip-of-service] [--type=type]` | Duplicate and expose a replication controller, service, or pod as a new Kubernetes service.
+`get`		| `kubectl get [(-o|--output=)json|yaml|wide|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=...] (TYPE [NAME | -l label] | TYPE/NAME ...) [flags]` | List one or more resources.
+`label`		| `kubectl label [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--resource-version=version]` | Add or update the labels of a resource.
+`logs`		| `kubectl logs [-f] [-p] POD [-c CONTAINER]` | Print the logs for a container in a pod.
+`namespace`	| `kubectl namespace [namespace]` | SUPERSEDED: Set and view the current Kubernetes namespace.
+`patch`		| `kubectl patch (-f FILENAME | TYPE NAME) -p PATCH` | Update the field(s) of a resource using stdin.
```

Comment:
Suggest we keep "Update the field(s) of a resource using strategic merge patch."

-p for --patch should be deprecated, so 
`kubectl patch (-f FILENAME | TYPE NAME | TYPE/NAME) --patch PATCH [flags]`